bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/PwaManifestServlet.java

```diff
@@ -147,12 +115,11 @@ private JsonArray getIcons(final Configuration configuration) {
                         final String path = p.get(PN_ICON_PATH, String.class);
 
                         json.addProperty(KEY_ICON_SRC, path);
-                        json.addProperty(KEY_ICON_SIZE, p.get(PN_ICON_SIZE, String.class));
+                        json.addProperty(KEY_ICON_SIZE, StringUtils.join(p.get(PN_ICON_SIZES, String[].class), " "));
                         json.addProperty(KEY_ICON_TYPE, mimeTypeService.getMimeType(path));
```

Comment:
figured chrome always takes png extensions else throws an  error in the browser console. So we can add a description to the configuration widget to enter png type assets.